### DISCUSSION TITLE
Getting the word count

### DISCUSSION THREAD

**Discussion by 2color** [znck/grammarly#271] [Q&A]
Hey there,

I've been using this wonderful plugin for almost two years now and it's immensely helpful!.

At some point I remember that it was possible to hover over the plugin in the status bar to get the current word count. Is this still possible? I'm using `v0.23.2` and couldn't find this.

Thank again for making this plugin Rahul ❤️

**Comment by znck** [comment]
Yes, will get it back. #273
